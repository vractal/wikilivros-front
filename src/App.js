import React, { Component } from 'react'
import { Provider } from 'react-redux'
import { Provider as CroodsProvider } from 'croods'
import { MuiThemeProvider } from '@material-ui/core/styles'
import './App.css'
import theme from './theme'

import store from './store/store'

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <CroodsProvider baseUrl={process.env.REACT_APP_API_URL}>
          <MuiThemeProvider theme={theme}>
            <div>New React App</div>
          </MuiThemeProvider>
        </CroodsProvider>
      </Provider>
    )
  }
}

export default App
