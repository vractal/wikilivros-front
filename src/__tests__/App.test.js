import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'

jest.mock('croods', () => ({
  createReducer: () => (state = {}, action = {}) => state,
  Provider: ({ children, ...props }) => (
    <div {...props}>Provider - {children}</div>
  ),
  Info: ({ children, ...props }) => <div {...props}>Info - {children}</div>,
}))

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot()
})
