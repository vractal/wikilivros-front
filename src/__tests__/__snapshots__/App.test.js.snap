// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  Provider - 
  <div>
    New React App
  </div>
</div>
`;
